simpinkscr/svg_to_simp_ink_script.py

```diff
@@ -1022,9 +1022,15 @@ def convert_guide(self, node):
         if color is not None:
             extra = ', color=%s' % repr(color)
 
+        # Determine if we were given a label.
+        extra_label = ''
+        label = node.get('inkscape:label')
+        if label is not None:
+            extra_label = ', label=%s' % repr(label)
+
```

Comment:
Please append to `extra` instead of introducing a new variable.

Reply:
I updated the code, so the label - if existent - will be appended to `extra`.